taqueria-vscode-extension/src/lib/helpers.ts

```diff
@@ -276,6 +277,45 @@ export const inject = (deps: InjectedDependencies) => {
 		vscode.window.showInformationMessage(msg)
 			.then(_ => Promise.resolve()) as Promise<void>;
 
+	const logAllNestedErrors = (err: TaqVsxError | TaqError | Error | any, output: Output) => {
+		try {
+			if (!err) {
+				return;
+			}
+			if (!shouldOutput(OutputLevels.error, output.logLevel)) {
```

Comment:
We can do `if (!err || !shouldOutput(OutputLevels.error, output.logLevel)) return;` since the 2 conditions lead to the same execution.

Reply:
Thanks for the suggestion. The main reason I kept the two separate is to keep it clear for the reader why we are returning. 

- For line 282, we return because we have reached the end of nested errors, this is the break from recursive calls.
- For line 285, we return because the configured level filters out this error.

So while both just `return`, one prevents the output from showing, and another short circuits because there is nothing to output.

I do not insist, by the way, to make such distinctions in Taqueria's code.

Reply:
I'll second that @AlirezaHaghshenas!